src/cljs/main/broadfcui/common/icons.cljs

```diff
@@ -81,4 +81,4 @@
      {:size 28})
    :render
    (fn [{:keys [props]}]
-     (icon {:style {:color "white" :fontSize (:size props)}} :warning-triangle))})
+     (icon {:style (merge {:color "white" :fontSize (:size props)} props)} :warning-triangle))})
```

Comment:
I don't think you want to change this line at all. See my comment where you're using this below.